main.go

```diff
@@ -96,13 +108,38 @@ func main() {
 
 	flag.IntVar(&restConfigBurst, "kube-api-burst", 30,
 		"Maximum number of queries that should be allowed in one burst from the controller client to the Kubernetes API server. Default 30")
+	flag.StringVar(&tlsOptions.TLSMinVersion, "tls-min-version", "VersionTLS10",
+		"The minimum TLS version in use by the webhook server.\n"+
+			fmt.Sprintf("Possible values are %s.", strings.Join(cliflag.TLSPossibleVersions(), ", ")),
+	)
+
+	flag.StringVar(&tlsOptions.TLSMaxVersion, "tls-max-version", "VersionTLS12",
```

Comment:
Maximum should be TLS 1.3 as well to allow server/client to choose between 1.2 and 1.3 by default.